assets/js/contact.js

```diff
@@ -9,8 +20,16 @@ function verifyCanSubmitEntry() {
   const piiError = document.getElementById('pii-warning');
   const piiErrorText = document.getElementById('pii-warning-message');
   const descriptionInput = document.getElementById('description');
+  const emailInput = document.getElementById('email');
+  const spamEmailDigests = (form.dataset.spamEmailAddresses || '').split(',').filter(Boolean);
+
   let alreadyAttemptedSubmission = false;
   form.addEventListener('submit', (event) => {
+    if (emailIsSpam(emailInput.value, spamEmailDigests)) {
```

Comment:
Nit: Stylistically I'd expect a boolean method/variable to be named with `is` as the prefix. I also think maybe we don't need the function at all, since it's unnecessary indirection that's not any more clear?

```suggestion
    if (spamEmailDigests.includes(sha256(emailInput.value))) {
```

Reply:
since the sha256 is async we'd have to do `await`, is this stylistically ok?
```suggestion
    if (spamEmailDigests.includes(await sha256(emailInput.value))) {
```